packages/remix/src/build.ts

```diff
@@ -467,10 +463,18 @@ module.exports = config;`;
     }),
   ]);
 
-  const output: BuildResultV2Typical['output'] = staticFiles;
+  const publicPath = remixConfig.publicPath.replace(/^\/|\/$/g, '');
+  const transformedBuildAssets = rename(buildAssets, name => {
+    return posix.join(publicPath, name);
```

Comment:
Could simplify this a bit.

```suggestion
  const transformedBuildAssets = rename(buildAssets, name => {
    return posix.join('./', remixConfig.publicPath, name);
```

Reply:
Done, thanks!